tests/event-log/event-log-level.spec.ts

```diff
@@ -0,0 +1,130 @@
+import chaiAsPromised from 'chai-as-promised';
+import { computeCid } from '../../src/utils/cid.js';
+import { EventLogLevel } from '../../src/event-log/event-log-level.js';
+import { TestDataGenerator as TDG } from '../utils/test-data-generator.js';
```

Comment:
NIT: as much as I appreciate why you did this, I personally try to avoid renaming things like this as it makes it harder to do a search (e.g. `grep`) for things in a codebase 😅

Reply:
good point! changed.